includes/Modules/Analytics_4.php

```diff
@@ -122,6 +122,8 @@ function( $old_value, $new_value ) {
 				$this->get_method_proxy( 'replicate_analytics_sharing_settings' )
 			);
 		}
+
+		add_action( 'googlesitekit_allow_tracking_disabled', $this->get_method_proxy( 'filter_analytics_allow_tracking_disabled' ) );
```

Comment:
It's a small detail, but this should be `add_filter`, not `add_action`.

```suggestion
		add_filter( 'googlesitekit_allow_tracking_disabled', $this->get_method_proxy( 'filter_analytics_allow_tracking_disabled' ) );
```